webrtc.html

```diff
@@ -5523,25 +5413,6 @@ <h2>Methods</h2>
                     </li>
                   </ol>
                 </li>
-                <li>
-                  <p>A track could have contents that are inaccessible to the
-                  application. This can be due to being marked with a
-                  <code>peerIdentity</code> option or anything that would make
-                  a track <a data-cite=
-                  "!fetch#concept-cors-check">
-                  CORS cross-origin</a>. These tracks can be supplied to the
-                  <code>addTrack</code> method, and have an
-                  <code><a>RTCRtpSender</a></code> created for them, but
-                  content MUST NOT be transmitted, unless they are also marked
-                  with <code>peerIdentity</code> and they meet the requirements
-                  for sending (see <a>isolated stream</a>).
-                  </p>
-                  <p>All other tracks that are not accessible to the
-                  application MUST NOT be sent to the peer, with silence
```

Comment:
It's important not to lose the cross-origin language here.

I suggest a much smaller change s/`being marked with a <code>peerIdentity</code> option or `//

Reply:
+1

Reply:
I left some of the text. I still think this is unnecessary and not well specified but we can improve this as a follow-up